docs/UserGuide.md

```diff
@@ -232,7 +232,19 @@ Example:`t/12-A nursing home`
 
 ### `d/TASK_DESCRIPTION | DATE TIME | INTERVAL TIME_PERIOD`
 
-_To be added_
+* `TASK_DESCRIPTION` can be any non-empty string made of alphanumeric characters.
+* `DATE TIME` must be of the form dd-MM-yy HHmm, but the time is optional. <br>
+  e.g. `2-7-22 1345`, `09-4-22`, `08-06-22 0900`, `7-06-22 2130` and `28-10-22` are all valid `DATE TIME`.
+* If `TIME` is omitted, the task will be created with a default time of `0000` hours.
```

Comment:
This point and everything below except the `TIME_PERIOD` bullet should be in the features section. Shaugn has already filled this part quite nicely so I suggest fixing the conflicts